tools/clang/tools/dxcompiler/dxcapi.cpp

```diff
@@ -37,10 +37,6 @@ DEFINE_CROSS_PLATFORM_UUIDOF(IDxcContainerBuilder)
 DEFINE_CROSS_PLATFORM_UUIDOF(IDxcOptimizerPass)
 DEFINE_CROSS_PLATFORM_UUIDOF(IDxcOptimizer)
 DEFINE_CROSS_PLATFORM_UUIDOF(IDxcRewriter)
-DEFINE_CROSS_PLATFORM_UUIDOF(IUnknown)
-DEFINE_CROSS_PLATFORM_UUIDOF(INoMarshal)
-DEFINE_CROSS_PLATFORM_UUIDOF(IStream)
-DEFINE_CROSS_PLATFORM_UUIDOF(ISequentialStream)
```

Comment:
Why aren't all of them moved to WinAdapter.cpp? Because they aren't windows classes I guess?

Reply:
Ideally, we want each interface to have `DECLARE_CROSS_PLATFORM_UUIDOF` in the header file and the cpp file that implements the interface to have `DEFINE_CROSS_PLATFORM_UUIDOF`. Until now we didn't have a home for these few interfaces because we didn't have a .cpp file for the adapter classes. So we had just placed them along with the others in dxcapi.cpp. WinAdapter.cpp is a better home for them I believe.